internal/cluster/transition_test.go

```diff
@@ -1443,7 +1459,7 @@ var _ = Describe("Refresh Cluster - With DHCP", func() {
 		mockHostAPI = host.NewMockAPI(ctrl)
 		mockMetric = metrics.NewMockAPI(ctrl)
 		clusterApi = NewManager(getDefaultConfig(), getTestLog().WithField("pkg", "cluster-monitor"), db,
-			mockEvents, mockHostAPI, mockMetric, nil, nil)
+			mockEvents, mockHostAPI, mockMetric, nil, nil, getOcsConfig())
```

Comment:
@filanov, is the mocking of code required here also? Because for transitions_test.go , it will require the actual code of operator Validations to execute.